conf.py

```diff
@@ -16,11 +16,23 @@
 project = 'Odoo'
 copyright = 'Odoo S.A.'
 
-# `version` if the version info for the project being documented, acts as replacement for |version|,
+# `version` is the version info for the project being documented, acts as replacement for |version|,
 # also used in various other places throughout the built documents.
 # `release` is the full version, including alpha/beta/rc tags. Acts as replacement for |release|.
 version = release = '14.0'
 
+# `current_release` is the version that was last released when the current branch was frozen. It has
+# the same value as `version` on release branches (XX.0) and has the value of the release branch's
+# `version` on SaaS branches and on master.
+# `previous_release` is the release before `current_release`.
+# `next_release` is the release coming after `current_release`.
+# `latest_release` is the version that was last released when the documentation was built. Similarly
+# to `current_release`, it does not take SaaS branches into account.
+current_release = '14.0'
+previous_release = '13.0'
+next_release = '15.0'
```

Comment:
> Also what happens for branches where the next release doesn't exist yet, is there still a next_release value with the future version or is there a synthetic value?

I'm actually considering removing that placeholder because I didn't find any usage for it in 14.0. WDYT @jcs-odoo?

If we keep it, I believe it would only be used to mention existing subsequent releases. For example, for stating that some feature is only available as of Odoo xx.0, which will would never write in the branch of the latest release or in master.

Reply:
True. It's best avoiding versions in the text whenever possible, anyway